docs/docs/dashboards.md

```diff
@@ -0,0 +1,268 @@
+```
+
+[Example documentation](../examples/dashboard_gzipped/readme).
+
+### URL
+
+Probably the easiest way to get started to add dashboards to your Grafana instances.
+You can use a GitHub URL.
+
+```yaml
+apiVersion: grafana.integreatly.org/v1beta1
+kind: GrafanaDashboard
+metadata:
+  name: grafanadashboard-from-url
+spec:
+  instanceSelector:
+    matchLabels:
+      dashboards: "grafana"
+  url: "https://raw.githubusercontent.com/grafana-operator/grafana-operator/master/examples/dashboard_from_url/dashboard.json"
+```
+
+Or why not use a finished dashboard from grafana.
+
+```yaml
+apiVersion: grafana.integreatly.org/v1beta1
+kind: GrafanaDashboard
+metadata:
+  name: grafanadashboard-from-grafana
+spec:
+  instanceSelector:
+    matchLabels:
+      dashboards: "grafana"
+  url: "https://grafana.com/api/dashboards/1860/revisions/30/download"
+```
+
+[Example documentation](../examples/dashboard_from_url/readme).
+
+### Jsonnet
+
+```yaml
+---
+apiVersion: grafana.integreatly.org/v1beta1
+kind: GrafanaDashboard
+metadata:
+  name: grafanadashboard-jsonnet
+spec:
+  resyncPeriod: 30s
+  instanceSelector:
+    matchLabels:
+      dashboards: "grafana"
+  jsonnet: |
+   local grafana = import 'grafonnet/grafana.libsonnet';
+   local dashboard = grafana.dashboard;
+   local row = grafana.row;
+   local singlestat = grafana.singlestat;
+   local prometheus = grafana.prometheus;
+   local template = grafana.template;
+
+   dashboard.new(
+     'JVM',
+     tags=['java'],
+   )
+   .addTemplate(
+     grafana.template.datasource(
+       'PROMETHEUS_DS',
+       'prometheus',
+       'Prometheus',
+       hide='label',
+     )
+   )
+   .addTemplate(
+     template.new(
+       'env',
+       '$PROMETHEUS_DS',
+       'label_values(jvm_threads_current, env)',
+       label='Environment',
+       refresh='time',
+     )
+   )
+   .addTemplate(
+     template.new(
+       'job',
+       '$PROMETHEUS_DS',
+       'label_values(jvm_threads_current{env="$env"}, job)',
+       label='Job',
+       refresh='time',
+     )
+   )
+   .addTemplate(
+     template.new(
+       'instance',
+       '$PROMETHEUS_DS',
+       'label_values(jvm_threads_current{env="$env",job="$job"}, instance)',
+       label='Instance',
+       refresh='time',
+     )
+   )
+   .addRow(
+     row.new()
+     .addPanel(
+       singlestat.new(
+         'uptime',
+         format='s',
+         datasource='Prometheus',
+         span=2,
+         valueName='current',
+       )
+       .addTarget(
+         prometheus.target(
+           'time() - process_start_time_seconds{env="$env", job="$job", instance="$instance"}',
+         )
+       )
+     )
+   )
+
```

Comment:
```suggestion
```
```

Reply:
Nevermind this suggestion, code block tags don't play nice with github suggestions, you might have to add it manually :sweat: